docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -164,6 +164,46 @@ Parameter explanation:
 
 The operation dereferences a null object and throws a NullPointerException. Review your code and add a null check.
 
+## UnimplementedTypeRule#
+
+UnimplementedTypeRule detects abstract classes and interfaces that are non-global and missing implementations or extensions.
+
+To invoke graph-based analysis rules run: `scanner:run --engine sfge --projectdir MyDirectory`.
+
+### Definition
+
+UnimplementedTypeRule is a traditional static analysis rule where a violation occurs at a point in the code where the interface or abstract class is declared. It doesn’t use sources or sinks.
+
+### Interpreting UnimplementedTypeRule Results
+
+Match any violation message that you receive with this case to understand more about the violation.
+
+*Common Case*
+
+> Extend, implement, or delete %s %s
+
+Parameter Explanation:
+
+Because this abstract class or interface has no implementations or extensions, it can’t be instantiated. It’s unnecessary and can be deleted.
+
+### UnimplementedTypeRule Limitations
+
+Because UnimplementedType rule excludes `global` scoped classes from consideration, these classes are prevented from being thrown as false positives and aren’t false negatives.
+
+## Pilot Rules
+
+To run each Graph Engine pilot rule, include the ```--with-pilot``` flag in your request. 
+
+To run all Graph Engine rules and all pilot rules, run:
+
+```sfdx scanner:run:dfa --with-pilot --engine sfge --projectdir /project/dir --target /project/dir/target1```
+
+To run a specific category of rules including the pilot rules in that category, include the category and the ```--with-pilot``` flag.
+
+**Example**:
+
+```sfdx scanner:run:dfa --category “Performance” --with-pilot --engine sfge --projectdir /project/dir --target /project/dir/target1```
+
 ## MultipleMassSchemaLookupRule
```

Comment:
This header (and its children) need an additional `#` at the start so they display at the same level as UnusedMethodRule and its children.